src/components/Button.js

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+export default function Button(props) {
```

Comment:
hmm usually I see components like this

```js
export default function Button({htmlClass, data, children}) {
```

Maybe it's for you don't type `props` every time

Reply:
done